api/src/certification/scoring/infrastructure/repositories/scoring-configuration-repository.js

```diff
@@ -3,17 +3,19 @@ import * as areaRepository from '../../../../shared/infrastructure/repositories/
 import * as competenceRepository from '../../../../shared/infrastructure/repositories/competence-repository.js';
 import { V3CertificationScoring } from '../../domain/models/V3CertificationScoring.js';
 
-export const listByLocale = async ({ locale }) => {
+export const getMostRecentByDateAndLocale = async ({ locale, date }) => {
```

Comment:
Est-ce que c'est pareil?
```suggestion
export const getLatestByDateAndLocale = async ({ locale, date }) => {
```

Reply:
It is indeed !